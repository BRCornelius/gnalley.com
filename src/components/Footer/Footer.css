.footer-container {
    background-color: #f3e42e;
    bottom: 0;
    color: white;
    display: inline-flex;
    min-height: 3rem;
    justify-content: center;
    align-items: center;
    width: -webkit-fill-available;
    position: sticky;
}