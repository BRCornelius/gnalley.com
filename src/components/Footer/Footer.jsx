import React from 'react';
import './Footer.css';

export const Footer = () => <div className="footer-container">
    <h3>Nalley and Dew, APLC • 2450 Severn Ave • Suite 100 • Metairie, LA 70001<br />Phone: (504) 838-8188 | Fax: (504) 838-0008</h3>
</div>