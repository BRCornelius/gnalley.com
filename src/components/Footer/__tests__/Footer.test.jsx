import { render } from '@testing-library/react';
import React from 'react';
import { Footer } from '../Footer';

const { container } = render(<Footer />)

test('Should render correctly', () => {
    expect(container).toMatchSnapshot()
});
