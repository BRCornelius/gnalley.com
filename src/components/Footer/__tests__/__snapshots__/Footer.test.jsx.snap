// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div>
  <div
    class="footer-container"
  >
    <h3>
      Nalley and Dew, APLC • 2450 Severn Ave • Suite 100 • Metairie, LA 70001
      <br />
      Phone: (504) 838-8188 | Fax: (504) 838-0008
    </h3>
  </div>
</div>
`;
